sous/Starter.php

```diff
@@ -34,5 +33,10 @@ public static function installTheme() {
   $yaml = Yaml::dump($system_theme_yml);
   file_put_contents('web/profiles/contrib/sous/config/install/system.theme.yml', $yaml);
   file_put_contents('web/profiles/contrib/sous/sous.info.yml', '  - '.$composerRoot.PHP_EOL, FILE_APPEND | LOCK_EX);
+  // Remove contrib theme after theme generation.
+  shell_exec ("rm -rf web/themes/contrib/emulsify-drupal/");
+  shell_exec ("sed -i 's/sous-project/$composerRoot/' .lando.yml");
+  // Write config folder location.
+  shell_exec ("cat web/profiles/contrib/sous/assets/scaffold/default/additions-default.settings.txt >> web/sites/default/default.settings.php");
```

Comment:
Attempting to set the default config directory to the root of the project: see https://github.com/fourkitchens/sous-drupal-distro/blob/4.x-beta/assets/scaffold/default/additions-default.settings.txt 

This could probably be done a better way. The goal here is to remove this as a manually step post-install.

Reply:
agreed. Removed.